<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bocchi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>361</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/image/bocchi.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>270</y>
        <width>261</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/image/ryou.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/image/kita.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/image/nijika.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stock_page">
     <widget class="QTableWidget" name="tableStock">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>361</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStockFind">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineStockFind">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>170</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>144</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>输入要查找的股票代码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DetailStock">
     <widget class="QPushButton" name="btnBuy">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>买入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPredict">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>预测</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBuy">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>470</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinSell">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>470</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSell">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>卖出</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>451</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="userInfo">
     <widget class="QTableWidget" name="tableUserInfo">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>551</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>90</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>持有股票</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>账户资金：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLineEdit" name="linePassMod"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelPass">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineNameMod"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="labelMoney">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btnModName">
         <property name="text">
          <string>修改用户名</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="btnModPass">
         <property name="text">
          <string>修改密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMain"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
   <addaction name="actionUserInfo"/>
   <addaction name="actionLogout"/>
   <addaction name="separator"/>
   <addaction name="actionStocks"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionUserInfo"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>股票</string>
    </property>
    <addaction name="actionStocks"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>登出</string>
   </property>
  </action>
  <action name="actionUserInfo">
   <property name="text">
    <string>个人信息</string>
   </property>
  </action>
  <action name="actionStocks">
   <property name="text">
    <string>股票信息</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionMain">
   <property name="text">
    <string>主页</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
